spec/features/inputs/nested_select_input_spec.rb

```diff
@@ -185,6 +185,80 @@
       end
     end
   end
+  context "with filters"  do
+    before do
+      register_page(Country, false) {}
+      register_page(Region, false) {}
+      register_page(City, false) {}
+
+      register_form(Invoice, false) do |f|
+        f.input :city_id, as: :nested_select,
+                          level_1: { attribute: :country_id },
+                          level_2: {
+                            attribute: :region_id,
+                            filters: { name_contains: 'Met' }
+                            },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.